shell/platform/fuchsia/flutter/component.cc

```diff
@@ -218,11 +255,16 @@ Application::Application(
 
   // Compare flutter_jit_runner in BUILD.gn.
   settings_.vm_snapshot_data_path = "pkg/data/vm_snapshot_data.bin";
-  settings_.vm_snapshot_instr_path = "pkg/data/vm_snapshot_instructions.bin";
+  settings_.vm_snapshot_instr = []() {
```

Comment:
Nit: We are moving away from the path based settings fields. Can we also change the `vm_snapshot_data_path` and others to use the mapping callback (not executable of course)? It's fine if you want to attempt this in another patch as it is not related to the fix at hand.

Reply:
Will fix it in a follow up commit.